docs/api/cases/cases-api-create.asciidoc

```diff
@@ -0,0 +1,203 @@
+[[cases-api-create]]
+== Create case API
+++++
+<titleabbrev>Create case</titleabbrev>
+++++
+
+Creates a case.
+
+=== Request
+
+`POST <kibana host>:<port>/api/cases`
```

Comment:
We should also document the path that takes a space ID, and document that if the space is not provided in the URL, the default space is used.

```
POST <kibana host>:<port>/s/<space_id>/api/cases
```